### PR TITLE
Added explantion about connecting to a vm using "virtctl ssh"

### PR DESCRIPTION
**Pull request by isaacdorfman**

### CODE REVIEW

**Comment by isaacdorfman** [comment]
@dankenigsberg

**Comment by mazzystr** [comment]
@isaacdorfman  Please rebase and sign your commit.  See https://github.com/kubevirt/user-guide#sign-your-commits

File: docs/virtual_machines/accessing_virtual_machines.md [hunk 1]
```diff
@@ -92,6 +92,18 @@ ClusterRole using a RoleBinding that targets a single namespace.
...
 
+In order to connect to a vm, you need to inject a key into it, and after the key is injected you can connect to it with "virtctl ssh".
+
+### Connecting to a vm with virtctl after injecting your key
+
+```
+virtctl ssh <user_name>@vm/<vm_name>.<namespace_name>
+```
+
+*Please not that "virtctl ssh" is only supported in up to date versions of virtctl.
```
Comment (mazzystr) [inline docs/virtual_machines/accessing_virtual_machines.md:103]: Please use an admonition here.  They make important note pop and grab the eye.

Example code : https://github.com/kubevirt/user-guide/blame/main/docs/virtual_machines/startup_scripts.md#L55-L56
Example render : http://kubevirt.io/user-guide/virtual_machines/startup_scripts/

**Review by mazzystr — CHANGES_REQUESTED** [review]

**Comment by cwilkers** [comment]
@isaacdorfman This PR is awaiting action by you, and has gone stale, are you able to sign your commits and address @mazzystr 's feedback?

**Pull request closed without merging**